src/main/java/edu/greenblitz/robotName/subsystems/Arm/Elbow/SimulationElbow.java

```diff
@@ -72,6 +72,7 @@ public void updateInputs(ElbowInputsAutoLogged inputs) {
         inputs.position = elbowSimulation.getAngleRads();
         inputs.velocity = elbowSimulation.getVelocityRadPerSec();
         inputs.absoluteEncoderPosition = elbowSimulation.getAngleRads();
+        inputs.temperature = 0;
```

Comment:
Yoni - our motor temperature is not 0 Celsius nor Kelvin nor Farenheit

Reply:
When your motor is in the merhav of cyber😏

Reply:
> When your motor is in the merhav of cyber😏

didnt think about that